package.json

```diff
@@ -125,6 +124,7 @@
     "electron-log": "^4.1.1",
     "extract-zip": "^1.6.7",
     "fs-extra": "^7.0.0",
+    "got": "^10.7.0",
```

Comment:
This is the last minor version before [11.0.0](https://github.com/sindresorhus/got/releases/tag/v11.0.0) which, generally, has breaking changes (though I didn't investigate if any applied to our pretty minimal use of this lib). My own use within this PR will also work for `10.6.0`, I only bumped it bc it was "free" (no extra work needed) and keeps us a bit more up to date.